src/imitation/scripts/eval_policy.py

```diff
@@ -96,12 +101,19 @@ def eval_policy(
             venv = reward_wrapper.RewardVecEnvWrapper(venv, reward_fn)
             logging.info(f"Wrapped env in reward {reward_type} from {reward_path}.")
 
-        trajs = rollout.generate_trajectories(
-            expert.get_expert_policy(venv),
-            venv,
-            sample_until,
-            rng=rng,
-        )
+        policy = expert.get_expert_policy(venv)
+        if explore_kwargs is not None:
+            policy = ExplorationWrapper(
+                rollout._policy_to_callable(policy, venv),
```

Comment:
This is using a private method in `rollout` (prefixed with `_`). However it seems we also use it in `preference_comparisons.py` in order to use `ExplorationWrapper`.

One option is to remove the `_` in `rollout.py` -- it seems OK to expose this in the public API, and it's already documented.

I'm also thinking perhaps we should change `ExplorationWrapper` to accept `types.AnyPolicy` instead, and then call `policy_to_callable` internally? We'd still need to expose that method in public API as `ExplorationWrapper` is outside of `rollout`, but this seems better than pushing the responsibility onto the caller.

Reply:
Second idea sounds good to me, am implementing that.